content/en/docs/ambient/install/helm/index.md

```diff
@@ -1,17 +1,23 @@
 ---
 title: Install with Helm
-description: Install Istio in Ambient mode with Helm.
+description: Install Istio with support for ambient mode with Helm.
 weight: 4
 aliases:
   - /docs/ops/ambient/install/helm-installation
   - /latest/docs/ops/ambient/install/helm-installation
+  - /docs/ambient/install/helm-installation
+  - /latest/docs/ambient/install/helm-installation
 owner: istio/wg-environments-maintainers
 test: yes
 ---
 
-This guide shows you how to install Istio in ambient mode with Helm.
-Aside from following the demo in [Getting Started with Ambient Mode](/docs/ambient/getting-started/),
-we encourage the use of Helm to install Istio for use in ambient mode. Helm helps you manage components separately, and you can easily upgrade the components to the latest version.
+{{< tip >}}
+Follow this guide to install and configure an Istio mesh with support for ambient mode.
+If you are new to Istio, and just want to try it out, follow the
+[quick start instructions](/docs/ambient/getting-started) instead.
+{{< /tip >}}
+
+We encourage the use of Helm to install Istio for use in ambient mode. To allow controlled upgrades, the control plane and data plane components are packaged and installed separately. (Because the ambient data plane is split across [two components](/docs/ambient/architecture/data-plane), the ztunnel and waypoints, upgrades involve separate steps for these components.)
```

Comment:
https://github.com/istio/istio.io/issues/15584 again, though that FAQ doesn't know ambient exists, and I think it's actually possible the recommendation is istioctl for sidecars and Helm for ambient.  We'll let that process play out.

Reply:
https://github.com/istio/istio.io/issues/15584 again, though that FAQ doesn't know ambient exists, and it may even be the case the recommendation is istioctl for sidecars and Helm for ambient.  We'll let that process play out.